From 6196214011422607a61bf44964d6326ee9b325e9 Mon Sep 17 00:00:00 2001
From: Cristy <urban-warrior@imagemagick.org>
Date: Thu, 6 Jul 2017 06:43:32 -0400
Subject: [PATCH] [2/2] Enable heap overflow check for stdin for mpc files

Enabling seekable streams is required to ensure checking the blob size
works when an image is streamed on stdin.

origin: https://github.com/ImageMagick/ImageMagick/commit/529ff26b68febb2ac03062c58452ea0b4c6edbc1
bug: https://github.com/ImageMagick/ImageMagick/issues/556
bug-debian: https://bugs.debian.org/867896

(cherry picked from commit 529ff26b68febb2ac03062c58452ea0b4c6edbc1)
---
 coders/mpc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/coders/mpc.c b/coders/mpc.c
index f49abaaf0..3c8f7d6f7 100644
--- a/coders/mpc.c
+++ b/coders/mpc.c
@@ -1006,7 +1006,6 @@ ModuleExport size_t RegisterMPCImage(void)
   entry=SetMagickInfo("CACHE");
   entry->description=ConstantString("Magick Persistent Cache image format");
   entry->module=ConstantString("MPC");
-  entry->seekable_stream=MagickTrue;
   entry->stealth=MagickTrue;
   (void) RegisterMagickInfo(entry);
   entry=SetMagickInfo("MPC");
